testing/hyperparams/test_space.py

```diff
@@ -89,3 +90,35 @@ def test_hyperparams_space_rvs_outputs_samples():
     for k, v in samples.iter_flat():
         assert k in space
         assert not isinstance(v, HyperparameterDistribution)
+
+
+def test_hyperparameter_samples_compress():
+    _flat_hps = {
```

Comment:
```suggestion
    _flat_hps = HyperparameterSamples({
```

Reply:
Will make sure to change it accordingly.